src/main/resources/db/changelog/changes/08-insert-user_roles-table.yml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: insert-user_roles
+      author: bohdan
+      changes:
+        - insert:
+            tableName: user_roles
+            columns:
+              - column:
+                  name: user_id
+                  value: 7
+              - column:
+                  name: role_id
+                  value: 2
+        - insert:
+            tableName: user_roles
+            columns:
+              - column:
+                  name: user_id
+                  value: 8
+              - column:
+                  name: role_id
+                  value: 1
```

Comment:
don't forget to leave last line empty
![image](https://github.com/Nonzeq/Online-Book-store/assets/49767049/ac3dfd8b-2d4e-46b6-9af3-8499e4ece139)